Dockerfile

```diff
@@ -43,7 +61,7 @@ RUN pip3 install ./scripts/diff_tester_export_insns
 
 RUN cd remill-build && \
     cmake --build . --target test_dependencies -- -j $(nproc) && \
-    CTEST_OUTPUT_ON_FAILURE=1 cmake --build . --verbose --target test -- -j $(nproc) && \
+    # CTEST_OUTPUT_ON_FAILURE=1 cmake --build . --verbose --target test -- -j $(nproc) && \
```

Comment:
This should be guarded by a check on the architecture. We use this in CI to test that the Docker build continues to work, at least for x86_64.

Reply:
Made this change in #695 👍